mods/key_value.js

```diff
@@ -77,7 +77,9 @@ KVBucket.prototype.createBucket = function(restbase, req) {
     if (!Array.isArray(opts.compression)) {
         opts.compression = [opts.compression];
     }
-    opts.updates = opts.updates || undefined;
+    opts.updates = {
```

Comment:
Do you see a use case for deviating from timeseries? These timeuuids should always be timestamp-ordered.